panel/src/components/Forms/Login.vue

```diff
@@ -78,23 +79,10 @@ export default {
         })
         .catch(response => {
           this.invalid   = true;
-          this.issue     = response.message;
+          this.issue     = this.$t("error.access.login");
```

Comment:
@bastianallgeier but this prevents displaying the actual error message which was the main concern of the issue, doesnt it?

Reply:
The original error message is still visible in the console in debug mode. In my opinion it should never end up in the alert.